src/main/java/uk/gov/hmcts/cmc/claimstore/models/otherparty/TheirDetails.java

```diff
@@ -43,6 +43,9 @@
     @NotNull
     private final Address address;
```

Comment:
Not related to this PR, but an address should be validated if provided.

Reply:
It is if you look 1 line up